### ISSUE TITLE
Add country flag emoji icons next to language in language-selector dropdown

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#7164]
### Describe the problem that you'd like solved

Add country flag emoji icons next to language in language-selector dropdown. Also, include the selected language flag by the selector.

Here are the flags for the countries currently in the list
🇨🇿 🇩🇪 🇺🇸 🇪🇸 🇫🇷 🇭🇷 🇵🇹 🇮🇳 🇺🇦 🇨🇳

Possible idea:
<img width="71" alt="Screen Shot 2022-11-17 at 11 15 10 AM" src="https://user-images.githubusercontent.com/978325/202537948-1d66adbe-4b96-444a-9d17-e21c07993d1a.png">

Before
<img width="153" alt="Screen Shot 2022-11-17 at 11 04 22 AM" src="https://user-images.githubusercontent.com/978325/202535627-86e1e3d5-ed3f-4b15-aec9-6b8a70f4e02c.png">

After
<img width="153" alt="Screen Shot 2022-11-17 at 11 17 50 AM" src="https://user-images.githubusercontent.com/978325/202538492-b15d523b-8294-40f7-8128-53064f7419b7.png">

**Comment by cdrini** [comment]
I don't think this is a good idea. Countries != languages; it's kind of frowned upon in the i18n space to do this. I don't want to see an American flag on my Open Library 😅 

eg https://weglot.com/blog/flags-to-represent-languages/

**Comment by mekarpeles** [comment]
Could we figure out flag from geo-loc? Is it worth it? This was a suggestion from a key project stakeholders, so I'd like to followup and hear thoughts + ideas prior to closing.

@MingdaMa maybe we should hold off on implementation for now (sorry!)

**Comment by tfmorris** [comment]
I agree this is an anti-pattern. "key project stakeholders" sounds like code for someone who doesn't have expertise but wants to throw their weight around anyway. 

Brazil, Mexico, Britain, French DOM/TOM are all good counter-examples. If you want it in PC-speak, you're also forcing former colonies to select the flag of their colonial oppressors.

**Comment by MingdaMa** [comment]
> 

Sure, no problem

**Comment by MingdaMa** [comment]
> Could we figure out flag from geo-loc? Is it worth it? This was a suggestion from a key project stakeholders, so I'd like to followup and hear thoughts + ideas prior to closing.
> 
> @MingdaMa maybe we should hold off on implementation for now (sorry!)

Sure, no problem

**Comment by andretchen0** [comment]
Associating language with a political flag is an anti-pattern.

- Living in a particular country doesn't mean you speak a particular language.
- Speaking a particular language doesn't mean you identify with the political flag of one big country where the language is spoken. (It may predispose you to having strong opinions about that country/flag, though!)

For reference, here are various multilingual government/promotional websites. Chosen at random. Not a single one uses flags for languages.

https://www.godominicanrepublic.com/
https://www.canada.ca/
https://www.belgium.be/en
https://www.exteriores.gob.es/en/Paginas/index.aspx
https://www.usa.gov/

**Comment by mekarpeles** [comment]
Enough valid concern has been raised by the community that I'm going to close this issue for now and report back to the stakeholder.

Thank you everyone.
@MingdaMa, is there another small project we may be able to work on together? Maybe this one? https://github.com/internetarchive/openlibrary/issues/5349
